runtime/doc/api.txt

```diff
@@ -1933,9 +1934,9 @@ nvim_get_option({name})                                    *nvim_get_option()*
         Option value (global)
 
 nvim_get_option_info({name})                          *nvim_get_option_info()*
-    Gets the option information for one option
+    Gets the information of an option
 
-    Resulting dictionary has keys:
+    The resulting dictionary has the following keys:
```

Comment:
nope. Don't want "the following" everywhere, it is obvious and clear the way it was, with reduced verbosity. This lets the reader focus on the important parts.